Dockerfile

```diff
@@ -110,6 +111,10 @@ RUN wget ${WGET_ARGS} https://github.com/zephyrproject-rtos/sdk-ng/releases/down
 	sh "zephyr-sdk-${ZSDK_VERSION}-x86_64-linux-setup.run" --quiet -- -d /opt/toolchains/zephyr-sdk-${ZSDK_VERSION} && \
 	rm "zephyr-sdk-${ZSDK_VERSION}-x86_64-linux-setup.run"
 
+RUN wget ${WGET_ARGS} https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v${ZSDK_ALT_VERSION}/zephyr-sdk-${ZSDK_ALT_VERSION}-x86_64-linux-setup.run && \
```

Comment:
if this is going to something we will change often, I would put it at the bottom  (after bsim), this way we only change minimal layers, maybe move both sdk lines below, because those change most often I guess.

Reply:
Changes made to move SDK and this down towards the end.